components/form/builder/README.md

```diff
@@ -0,0 +1,23 @@
+# FormBuilder
```

Comment:
FormBuilder seems to be very very generic and I think this form is so specific because it works in a specific way by chaining fields, each one depending on the value selected from the previous one.

Reply:
Under team discussion...

Reply:
No more discussion 😶